providers/os/resources/os.lr

```diff
@@ -929,6 +931,28 @@ iptables.entry {
   chain string
 }
 
+fstab @defaults("path") {
+  init(path? string)
+  path string
+
+  entries() []fstab.entry
+}
+
+private fstab.entry @defaults("device mountpoint") {
+  // Device referenced in the fstab, e.g. LABEL=rootfs
+  device string
+  // Mountpoint, e.g. '/'
+  mountpoint string
+  // Filesystem type, e.g. ext4
+  fstype string
+  // Mount options, e.g. defaults
+  options string
+  // Dump frequency, e.g. 0
```

Comment:
```suggestion
  // Dump frequency, e.g., 0
```

Reply:
In what units?

Reply:
doesn't have any
> The dump level (any integer). A level 0, full backup, specified by -0 guarantees the entire file system is copied (but see also the -h option below). A level number above 0, incremental backup, tells dump to copy all files new or modified since the last dump of a lower level. The default level is 0. Historically only levels 0 to 9 were usable in dump, this version is able to understand any integer as a dump level.